src/main/java/gregtech/common/metatileentities/storage/MetaTileEntityQuantumChest.java

```diff
@@ -345,8 +381,10 @@ public void writeInitialSyncData(PacketBuffer buf) {
         super.writeInitialSyncData(buf);
         buf.writeByte(getOutputFacing().getIndex());
         buf.writeBoolean(autoOutputItems);
+        this.virtualItemStack.setCount(1);
```

Comment:
this was originally to ensure that the stack size of the virtual item stack is always 1, but that's now handled by the inserting logic of the chest handler, so this could be removed.